plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -548,9 +549,13 @@ private void testSelectOrPartitionedByTimestampWithTimeZone(boolean partitioned)
                         "('_timestamptz', NULL, NULL, 0e0, NULL, '2021-10-31 00:30:00.005 UTC', '2021-10-31 00:30:00.007 UTC'), " +
                         "(NULL, NULL, NULL, NULL, 3e0, NULL, NULL)");
 
+        // Disable default filter factor to avoid getting stats when NDV is not present
```

Comment:
Could we update tests cases or is it crucial to have have stats missing?

Reply:
iceberg doesn't have NDV stats, I don't want that covered up by filter factor, keeping this as is